packages/mobile/metro.config.js

```diff
@@ -7,6 +7,20 @@
 
 const blacklist = require('metro-config/src/defaults/exclusionList');
 
+const path = require('path');
+
+const watchFolders = [
+  // path.resolve(__dirname, '../identity'),
+  // path.resolve(__dirname, '../nectar'),
+  path.resolve(__dirname, '../waggle')
+]
+
+const extraNodeModules = {
+  // '@quiet/identity': path.resolve(__dirname, '../identity'),
+  // '@quiet/nectar': path.resolve(__dirname, '../nectar'),
+  '@quiet/waggle': path.resolve(__dirname, '../waggle')
```

Comment:
mobile app imports waggle? I thought it uses it as zipbundle?

Reply:
It was before I introduced zipbundling waggle. I'm about to remove it